src/Tribe/Views/V2/Views/Traits/HTML_Cache.php

```diff
@@ -368,28 +379,16 @@ protected function get_view_nonce_json_properties() {
 	 * the correct string placeholder so we can remove it later.
 	 *
 	 * @since 5.0.0
+	 * @since TBD Deprecating for new nonce structure.
+	 *
+	 * @deprecated
 	 *
 	 * @param string $html HTML with the nonces to be replaced.
 	 *
 	 * @return string  HTML after replacement is complete.
 	 */
 	protected function extract_nonces_before_cache( $html ) {
-		$nonce_fields = $this->get_view_nonce_fields();
-		$nonce_attrs  = $this->get_view_nonce_attributes();
-		$nonce_props  = $this->get_view_nonce_json_properties();
-
-		foreach ( $nonce_fields as $action => $field ) {
-			$html = preg_replace( '!(<input[^>]+name="' . preg_quote( $field, '!' ) . '"[^>]+value=")[^"]*("[^>]*>)!', '\1%%NONCE:' . $action . '%%\2', $html );
-			$html = preg_replace( '!(<input[^>]+value=")[^"]*("[^>]+name="' . preg_quote( $field, '!' ) . '"[^>]*>)!', '\1%%NONCE:' . $action . '%%\2', $html );
-		}
-
-		foreach ( $nonce_attrs as $action => $attr ) {
-			$html = preg_replace( '!(' . preg_quote( $attr, '!' ) . '=")[^"]*(")!', '\1%%NONCE:' . $action . '%%\2', $html );
-		}
-
-		foreach ( $nonce_props as $action => $prop ) {
-			$html = preg_replace( '!("' . preg_quote( $prop, '!' ) . '":")[^"]*(")!', '\1%%NONCE:' . $action . '%%\2', $html );
-		}
+		_deprecated_function( __METHOD__, 'TBD' );
```

Comment:
What do you think about moving the deprecated methods to the end of the class? I find that helps keep them together and one day get removed.

Reply:
+1